x/emissions/metrics/metrics_util.go

```diff
@@ -44,10 +44,10 @@ func MeasureRpcRequestLatency(endpoint string, startTime time.Time) {
 // This metric counts the number of events produced by the system.
 // Metric Name:
 //
-//	allora_loadtest_produce_count
+//	allora_emissions_produce_count
 func IncrProducerEventCount(msgType string) {
 	telemetry.IncrCounterWithLabels(
-		[]string{"allora", "loadtest", "produce", "count"},
+		[]string{"allora", "emissions", "event", "count"},
```

Comment:
```suggestion
		[]string{"allora", "emissions", "event", "total"},
```

A counter is usually suffixed by `_total` to differentiate it from the counter of an histogram